mac/expert/determine_processor_architecture.tw2

```diff
@@ -0,0 +1,32 @@
+::Determine processor architecture on a Mac - Expert
+# Determine processor architecture on a Mac
+
+To determine your processor architecture, run:
+
+```
+uname -m | if grep -q 64; then echo "64-bit CPU"; else echo "32-bit CPU"; fi
+```
+
+#### Verify
```

Comment:
We don't need the verify step for the experts path on this—the main thing is to give them the command to use.